src/logic/operate.js

```diff
@@ -0,0 +1,31 @@
+import Big from 'big.js'; // eslint-disable-line
+
+const operate = (num1, num2, operation) => {
+  const n = Big(num1);
+  let evaluation;
```

Comment:
Kindly add this code snippet after line 4.

``` const n2 = Big(num2) ```

Then replace num2 inside operate function with n2.